scripts/mkcloud

```diff
@@ -837,6 +837,10 @@ function install_vbmc
     # virtualenv is also not (yet) available in standard repos
     virtualenv --version 2> /dev/null || \
         rpm -i http://download.suse.de/ibs/SUSE:/SLE-12-SP4:/Update:/Products:/Cloud9/standard/noarch/python-virtualenv-15.1.0-3.3.noarch.rpm
+    # subprocess32 requires gcc to build the package in venv
+    # install pre-build package globally
+    rpm -q python-subprocess32 > /dev/null || \
+        rpm -i http://download.suse.de/ibs/SUSE:/SLE-12-SP4:/Update:/Products:/Cloud9/standard/x86_64/python-subprocess32-3.5.1-1.14.x86_64.rpm
```

Comment:
This requires going to the Nürnberg download source even if this is running on the other side of the world. Why not use the existing repo management logic to get the right download host and add the Cloud9 update repo and then install with zypper like normal?

Reply:
I agree 100% but I have no idea where this script would be executed and if custom repo would cause any unexpected side effects. @bmwiedemann @jdsn wdyt?

Reply:
@cmurphy my biggest concern is that installing the virtualbmc rpm would pull `python-cryptography` v2.3.1 while currently there's v1.3.1 installed. Also note that these packages will be installed only once on the hosts (not with every build) not to mention that `SUSE:SLE-12-SP4:Update:Products:Cloud9` doesn't seem to be mirrored to clouddata (or I just couldn't find it there).